Extensions/BitmapText/bitmaptextruntimeobject-pixi-renderer.js

```diff
@@ -0,0 +1,147 @@
+/**
+ * The PIXI.js renderer for the Bitmap Text runtime object.
+ *
+ * @class BitmapTextRuntimeObjectPixiRenderer
+ * @constructor
+ * @param {gdjs.BitmapTextRuntimeObject} runtimeObject The object to render
+ * @param {gdjs.RuntimeScene} runtimeScene The gdjs.RuntimeScene in which the object is
+ */
+gdjs.BitmapTextRuntimeObjectPixiRenderer = function(
+  runtimeObject,
+  runtimeScene
+) {
+  this._object = runtimeObject;
+
+  const style = new PIXI.TextStyle();
+  style.fontFamily = runtimeScene
+    .getGame()
+    .getFontManager()
+    .getFontFamily(runtimeObject._fontFamily);
+  style.fontSize = runtimeObject._fontSize;
+  style.wordWrap = runtimeObject._wordWrap;
+  style.wordWrapWidth= runtimeObject._wrappingWidth;
+  style.fill = runtimeObject._color;
+
+  const slugFontName =
+    style.fontFamily + '-' + style.fontSize + '-' + style.fill + '-bitmapFont';
+
+  if (!PIXI.BitmapFont.available[slugFontName]) {
+    PIXI.BitmapFont.from(slugFontName, style, {
+      chars: PIXI.BitmapFont.ASCII,
+      textureWidth: 1024,
+    });
+  }
+
+  // Load (or reset) the text
+  if (this._pixiObject === undefined) {
+    this._pixiObject = new PIXI.BitmapText('BitmapText object', {
+      fontName: slugFontName,
+    });
+
+    this._pixiObject.align = runtimeObject._align;
+
+    this.style = style;
+    this.constructorSlugFontName = slugFontName;
```

Comment:
The style and slug should be marked as private - you don't want the style to be directly modified without setting dirty flag